addons/website_slides/views/website_slides_templates_utils.xml

```diff
@@ -96,4 +96,24 @@
     </a> to download resources
 </template>
 
+<!-- Python equivalent of the JS template "website.slides.sidebar.done.button" -->
+<template id="slide_sidebar_done_button" name="Sidebar Done Button">
+    <div t-if="is_member"
+        class="o_wslides_sidebar_done_button"
+        t-att-data-id="slide.id"
+        t-att-data-completed="1 if slide_completed else 0"
+        t-att-data-can-self-mark-completed="int(slide.can_self_mark_completed)"
+        t-att-data-can-self-mark-uncompleted="int(slide.can_self_mark_uncompleted)"
```

Comment:
Same remark here, should support True/False for those 3 attributes?
(If it does it for "is_member", it should do it for those).